### ISSUE TITLE
Write a join strategy for updating files based on modification date (or other criteria)

### ISSUE THREAD

**Issue by jtapolczai** [jtapolczai/FileSync#12]
For entries which are present in both trees, we should be able to overwrite one or the other based on a predicate

```
a -> IO TreeDiff -- left only: overwhrite the left with the right; right only: analogous; both: no overwrite
```

Ideally, it should be freely combineable with other strategies.

**Comment by jtapolczai** [comment]
This is handled via the

```haskell
a -> IO (Maybe FileAction)
```

handlers in summaryJoin now.

**Comment by jtapolczai** [comment]
Closed by 4ad57b3f2832ee2bfa4791d9b68f14ab569a7dc4.
